docs/use-cases/headless-cms/security.md

```diff
@@ -0,0 +1,109 @@
+---
+description:
+  This guide covers best practices for security, authentication, and permissions within the context of Headless CMS.
+tags: []
+skill_level:
+directus_version:
+author_override:
+author: Bryant Gillespie
+---
+
+# Security in Headless CMS
+
+> {{ $frontmatter.description }}
+
+[Headless CMS](/use-cases/headless-cms/introduction) as an architecture choice is generally more secure than Traditional
+CMS because your content is separated from the presentation layer. Given that users don't directly interact with a
```

Comment:
> [Headless CMS](/use-cases/headless-cms/introduction) as an architecture choice is generally more secure than Traditional CMS because your content is separated from the presentation layer.

This only holds true if the frontend is Statically Generated and does not directly interact with the headless cms. If the frontend does direct API calls to the Directus instance then there is no security advantage anymore. This is an advantage of static site generators not directly of headless cms's per se.

Reply:
You're absolutely right. Maybe like this: "Headless CMS, when coupled with statically-generated sites, are architecture choices that are generally more secure..."